configs/recognition/swin/swin-large_p244-w877-in22k-pre_16xb8-amp-32x2x1-30e_kinetics700-rgb.py

```diff
@@ -0,0 +1,100 @@
+_base_ = [
+    'swin-large_p244-w877-in22k-pre_8xb8-amp-32x2x1-30e_kinetics400-rgb.py'
+]
+
+model = dict(cls_head=dict(num_classes=700))
+
+dataset_type = 'VideoDataset'
+data_root = 'data/kinetics700/videos_train'
+data_root_val = 'data/kinetics700/videos_val'
+ann_file_train = 'data/kinetics700/kinetics700_train_list_videos.txt'
+ann_file_val = 'data/kinetics700/kinetics700_val_list_videos.txt'
+ann_file_test = 'data/kinetics700/kinetics700_val_list_videos.txt'
+
+file_client_args = dict(
```

Comment:
remove file_client_args

Reply:
I think it's better to reserve the `file_client_args` like [this](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/configs/_base_/datasets/coco_detection.py#L5-L14) .